src/components/analytics/ComparisonAnalysis.vue

```diff
@@ -144,14 +160,141 @@ export default defineComponent({
         ],
       };
     },
+    getCurrentTime(){
+			return new Date().toLocaleTimeString();
+		},
+		getCurrentDate(){
+			const current = new Date()
+      return current.toLocaleDateString()
+		},
     printBill(div_id: string, title: string) {
+      const styles = `
```

Comment:
use printDiv function created by khunshan.
Also import styles from constants (you can create new constant file inside this component)